wagtail/wagtailsearch/backends/base.py

```diff
@@ -64,8 +64,8 @@ def _process_filter(self, field_attname, lookup, value):
 
         if result is None:
             raise FilterError(
-                'Could not apply filter on search results: "' + field_attname + '__'
-                + lookup + ' = ' + text_type(value) + '". Lookup "' + lookup + '"" not recognosed.'
+                'Could not apply filter on search results: "' + field_attname + '__' +
+                lookup + ' = ' + text_type(value) + '". Lookup "' + lookup + '"" not recognosed.'
```

Comment:
'recognosed'? :-) (Not related to the PEP8 fixes I know, but worth fixing too I dare say...)